### ISSUE TITLE
Timezone inconsistency

### ISSUE THREAD

**Issue by ghost** [caktus/django-timepiece#503]
We aren't doing a good job of handling timezones.  I think that things work reliably when `settings.USE_TZ` is False.  I would like to make timezone handling explicit & document our choices so that users know what to expect.

This approach seems pretty straightforward:
1. Store everything in the database as UTC.  
2. Manipulate everything on the backend in UTC.  
3. When passing data to/from the front end, translate from `settings.TIME_ZONE` to UTC.  Where practical, make it clear on the front end what the time zone is.

The bottom line: Things work for our use case, by coincidence if not design. No immediate changes are needed but this is probably the most important clean up we can make.
